app/configurator/components/field.tsx

```diff
@@ -229,14 +229,14 @@ export const DataFilterSelectDay = ({
       onChange={fieldProps.onChange}
       name={dimensionIri}
       value={dateValue}
+      isDayDisabled={isDisabled}
       inputProps={{
         id,
         disabled,
       }}
       dayPickerProps={{
         fromMonth,
         toMonth,
-        disabledDays: isDisabled,
```

Comment:
I do not even understand why the day picker allows clicking on disabled days :/

Reply:
Thanks for the fix.

Reply:
Np!